runtime/vm/vmprops.c

```diff
@@ -1316,6 +1316,28 @@ containsBackslashU(const char *userString, UDATA stringLength) {
 	return FALSE; /* hit the end of the string without encountering \u sequence */
 }
 
+static U_8 *
+convertString(J9JavaVM *vm, I_32 fromCode, const char *userString, UDATA stringLength) {
+	PORT_ACCESS_FROM_JAVAVM(vm);
+	U_8 *result = NULL;
+	I_32 bufferLength = j9str_convert(fromCode, J9STR_CODE_MUTF8, userString, stringLength, NULL, 0);
+	if (bufferLength >= 0) {
+		U_8 *mutf8Buffer = NULL;
+		++bufferLength; /* account for terminating null */
+		mutf8Buffer = j9mem_allocate_memory(bufferLength, OMRMEM_CATEGORY_VM);
+		if (NULL != mutf8Buffer) {
+			I_32 resultLength = j9str_convert(fromCode, J9STR_CODE_MUTF8, userString, stringLength,
+					(char*)mutf8Buffer, bufferLength);
+			/* j9str_convert null-terminated the string */
+			if (resultLength >= 0) {
+				result = mutf8Buffer;
+			} else {
+				j9mem_free_memory(mutf8Buffer);
+			}
+		}
+	}
+	return result;
+}
 
 U_8*
 getMUtf8String(J9JavaVM * vm, const char *userString, UDATA stringLength) {
```

Comment:
Space before `*`, `{` on a separate line:
```c
U_8 *
getMUtf8String(J9JavaVM *vm, const char *userString, UDATA stringLength)
{
```

Reply:
Pushed the changes to address these review comments.